server/schedule/filter/filters.go

```diff
@@ -292,69 +326,105 @@ func (f *StoreStateFilter) Type() string {
 
 // conditionFunc defines condition to determine a store should be selected.
 // It should consider if the filter allows temporary states.
-type conditionFunc func(*config.PersistOptions, *core.StoreInfo) bool
+type conditionFunc func(*config.PersistOptions, *core.StoreInfo) plan.Status
 
-func (f *StoreStateFilter) isRemoved(opt *config.PersistOptions, store *core.StoreInfo) bool {
+func (f *StoreStateFilter) isRemoved(opt *config.PersistOptions, store *core.StoreInfo) plan.Status {
 	f.Reason = "tombstone"
-	return store.IsRemoved()
+	if store.IsRemoved() {
+		return tombstoneStatus
+	}
+	return okStatus
 }
 
-func (f *StoreStateFilter) isDown(opt *config.PersistOptions, store *core.StoreInfo) bool {
+func (f *StoreStateFilter) isDown(opt *config.PersistOptions, store *core.StoreInfo) plan.Status {
 	f.Reason = "down"
-	return store.DownTime() > opt.GetMaxStoreDownTime()
+	if store.DownTime() > opt.GetMaxStoreDownTime() {
+		return downStatus
```

Comment:
should we output the 'max-store-down-time'

Reply:
Do you mean the value of `max-store-down-time`?

Reply:
yep

Reply:
The detailed message can be improved later if necessary.